src/main/java/org/mockito/internal/junit/JUnitTestRule.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2020 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.junit;
+
+import org.junit.runner.Description;
+import org.junit.runners.model.Statement;
+import org.mockito.exceptions.base.MockitoException;
+import org.mockito.junit.MockitoTestRule;
+import org.mockito.plugins.MockitoLogger;
+import org.mockito.quality.Strictness;
+
+public final class JUnitTestRule extends AbstractJUnitRule implements MockitoTestRule {
+
+    private final Object testInstance;
+
+    public JUnitTestRule(MockitoLogger logger, Strictness strictness, Object testInstance) {
+        super(logger, strictness);
+        this.testInstance = testInstance;
+    }
+
+    @Override
+    public Statement apply(Statement base, Description description) {
+        if (description.isSuite()) {
+            throw new MockitoException("JUnitTestRule can not be used as a @ClassRule.");
```

Comment:
double checking - do we have coverage for this case?

Reply:
Done

Reply:
(Turns out you can only use `@ClassRule` on static fields, which doesn't work if you need to specify `this`.)